src/transformers/trainer.py

```diff
@@ -2413,17 +2482,21 @@ def save_model(self, output_dir: Optional[str] = None, _internal_call: bool = Fa
 
         Will only save from the main process.
         """
-
         if output_dir is None:
             output_dir = self.args.output_dir
 
         if is_torch_tpu_available():
             self._save_tpu(output_dir)
         elif is_sagemaker_mp_enabled():
             # Calling the state_dict needs to be done on the wrapped model and on all processes.
+            os.makedirs(output_dir, exist_ok=True)
             state_dict = self.model_wrapped.state_dict()
             if self.args.should_save:
                 self._save(output_dir, state_dict=state_dict)
+            if SMP_POST_1_10:
+                # 'user_content.pt' indicates model state_dict saved with smp >= 1.10
+                user_content = {"smp_ver : >= 1.10"}
+                torch.save(user_content, os.path.join(output_dir, "user_content.pt"))
```

Comment:
Why save a dictionary with `torch.save`? There are no tensors in this. Also you can just touch the file as it's not read in the tests before.

Reply:
Yeah, this is true!